Add translations of BP 54 to 64 by Mathieu POUCHES (#442)
cnumr/best-practices: chapters/BP_060_en.md

> ## Encode audio files outsides of the CMS

Reviewer: ```suggestion
## Encode audio files outside of the CMS
```

### Identifiers

| GreenIT | V2  | V3  |  V4  |
|:-------:|:---:|:---:|:----:|
|   96    | 60  | 60  |      |

### Categories

| Life cycle |    Tiers    | Responsible |
|:----------:|:-----------:|:-----------:|
|   5. Use   | User/Device |    User     |

### Indications

| Priority | Implementation difficulty | Ecological impact |
|:--------:|:-------------------------:|:-----------------:|
|    4     |             4             |         4         |

|                      Saved resources                      |
|:---------------------------------------------------------:|
|            Processor / RAM / Network / Storage            |

### Description

> Adapt and optimize audio files outside of the websites. This method saves a lot of bandwidth and relieves CPU and RAM as web servers are not optimized for (re)encoding audio files.

Reviewer: ```suggestion
Adapt and optimize audio files outside the websites. This method saves a lot of bandwidth and relieves CPU and RAM as web servers are not optimized for (re)encoding audio files.
```

### Examples

> If an audio file added directly in the CMS weighs 7 MB, it is possible to save:

Reviewer: ```suggestion
If an audio file added directly to the CMS weighs 7 MB, it is possible to save:
```

  - 7 MB of bandwidth at the upload;

>   - 100 KB when listening  t the resulting file;

Reviewer: ```suggestion
  - 100 KB when listening to the result file;
```
